Lib/test/test_py_compile.py

```diff
@@ -192,6 +192,14 @@ def test_invalidation_mode(self):
                 fp.read(), 'test', {})
         self.assertEqual(flags, 0b1)
 
+    def test_quiet(self):
+        bad_coding = os.path.join(os.path.dirname(__file__), 'bad_coding2.py')
+        with support.captured_stderr():
+            self.assertIsNone(py_compile.compile(bad_coding, doraise=False, quiet=2))
```

Comment:
We can drop `assertIsNone`s since the return value of `py_compile.compile()` isn't important.